MAINTAINERS.md

```diff
@@ -0,0 +1,32 @@
+# Maintainers
+
+This document provides maintenance procedures to assist maintainers.
+
+## Updates to the Envoy and Nighthawk dependency for Salvo
+
+Salvo reuses large parts of Envoy's and Nighthawk's build system and codebase, so keeping Salvo up
+to date with Envoy's and Nighthawk's changes is an important maintenance task. When performing the
+update, follow this procedure:
+
+1. Create a fork of Salvo, or fetch upstream and merge changes into your fork if you already have
+one.
+1. Create a new branch from `main`, e.g. `envoy-update`.
+1. Sync (copy) [.bazelrc](.bazelrc) from
```

Comment:
We have a potential problem here - we are syncing configuration files from two repositories. If we copy from both of those at head (the latest commit) they may not be in sync with each other.

Ideally we would:
1. Copy files we need from Nighthawk at head (latest commit).
2. Look at Nighthawks configuration and determine the Envoy commit Nighthawk is updated to, i.e. y looking [here](https://github.com/envoyproxy/nighthawk/blob/f911358d0e453180b3cc0cd8affe4b3319cfe6f9/bazel/repositories.bzl#L3).
3. Copy the files from Envoy at the commit from (2).

Reply:
I can understand what you worried about, we copy `.bashrc` from Nighthawk's one while copy `ci/run_envoy_docker.sh` and `ci/envoy_build_sha.sh` from Envoy's ones. However, as you suggested, if we look at Nighthawk's `repositories.bzl` to find out the Envoy commit, then copy the files from Envoy at this commit finally, we will lose Nighthawk's unique parts. 

So how about this following guide:
1. Look at Nighthawks configuration and determine the Envoy commit Nighthawk is updated to, i.e. y looking [here](https://github.com/envoyproxy/nighthawk/blob/f911358d0e453180b3cc0cd8affe4b3319cfe6f9/bazel/repositories.bzl#L3).
2. Copy the files from Envoy at the commit from (1).
3. Be sure to retain Nighthawk's modifications in `.bazelrc` from [here](https://github.com/envoyproxy/nighthawk/blob/f911358d0e453180b3cc0cd8affe4b3319cfe6f9/.bazelrc) all lines that are unique to Nighthawk are marked with comment `# unique`.
4. Be sure to retain Salvo's modification from [.bazelrc](.bazelrc), all lines that are unique to
   Salvo are marked with comment `# Salvo unique`.

Reply:
Maybe it's complicated for maintainers, I'm not sure if it's ok we sync files only from Envoy or only from Nighthawk? To be honest, I don't know much about their relationship and possible impact at the moment. Please tell me if I'm wrong.

Reply:
I think we should try to simplify this for files where we need to retain both Nighthawk's and Salvo's local modifications by copying them from Nighthawk instead of from Envoy. Looks like that only applies to `.bazelrc`. I.e. we can keep most of this as suggested in your last comment, but I would indicate to copy `.bazelrc` from Nighthawk.

Reply:
@mum4k  I updated it about the `.bazelrc` synchronization, I tried to keep it as simple as possible. Is it ok?

Reply:
Well, the reason is to resolve the problem you mentioned above: 
```
We have a potential problem here - we are syncing configuration files from two repositories. If we copy from both of those at head (the latest commit) they may not be in sync with each other.
```
My understanding is that if  we copy from `.bazelrc` from Nighthawk's one, and copy `ci/run_envoy_docker.sh` and `ci/envoy_build_sha.sh` from Envoy's ones. The `.bazelrc` file may not sync with `ci/run_envoy_docker.sh` and `ci/envoy_build_sha.sh`. So we need to find a way to confirm the `.bazelrc` file copied from Nighthawk is updated to Envoy's `.bazelrc`. 

On second thought, this is no longer a problem, if we look at "Envoy commit Nighthawk update to", they are always in sync. Sorry for making simple things complicated, I will update again to keep it simple.

Reply:
Updated: https://github.com/envoyproxy/envoy-perf/pull/140/commits/bd036c894e03bcda1aea991538f36d6cf6c2d5e2, please review again. Thanks.